app/src/test/java/com/aws/amazonlocation/viewmodel/explore/ExploreVMSearchSuggestionTest.kt

```diff
@@ -89,4 +90,77 @@ class ExploreVMSearchSuggestionTest : BaseTest() {
             cancelAndIgnoreRemainingEvents()
         }
     }
+    @Test
+    fun testMLatLngInitializedToNull() {
```

Comment:
I don't understand the purpose of these tests. it directly set a value to null and assert that value is null. what's the purpose?